sdk/src/main/java/com/horizen/account/transaction/AccountTransaction.java

```diff
@@ -24,5 +24,7 @@
 
     public abstract byte[] getData();
 
+    public abstract PR getRealSignature();
```

Comment:
It is a bit strange to see both `getRealSignature` and `getSignature` methods in the `AccountTransaction`. Let's have only `getSignature()`.

In EthTransaction we may rename `getRealSignature` to `getSignature`. And don't use the current `getSignature` at all. Instead we may define a method like `getSignatureData` that will return `Sign.SignatureData` and used in all the places needed: `toString`/RPC/serialization

Reply:
Ok, done